pkg/kudoctl/cmd/plan/plan_status.go

```diff
@@ -60,12 +60,17 @@ func status(kc *kudo.Client, options *Options, ns string) error {
 		return ""
 	}
 
-	rootDisplay := fmt.Sprintf("%s (Operator-Version: \"%s\" Active-Plan: \"%s\")", instance.Name, instance.Spec.OperatorVersion.Name, lastPlanStatus.Name)
+	rootDisplay := fmt.Sprintf("%s (Operator-Version: \"%s\" Active-Plan: \"%s\", last updated: \"%s\")", instance.Name, instance.Spec.OperatorVersion.Name, lastPlanStatus.Name, instance.Status.AggregatedStatus.LastUpdated.Format("2006-01-02 15:04:05"))
```

Comment:
`lastPlanStatus` isn't the aggregated status. Using `LastUpdated` of the aggregated status here might divert from what is captured with `lastPlanStatus`.

Reply:
I was kind of OK with the `LastUpdated` value from `AggregatedStatus` here, as long as it's updated to reflect when any other updates occur (as you've suggested and I've hopefully now captured).